customHttp.yml

```diff
@@ -0,0 +1,18 @@
+# file added for AWS amplify, setting security headers. See https://docs.aws.amazon.com/amplify/latest/userguide/custom-headers.html#example-security-headers
+# https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-chtml.js is added for MATH-jax support
+# font-src is added for MATH-jax support
+# script-src unsafe-inline is added because this is an SPA and won't work without it.
+# style-src unsafe-inline is added because of MUI and Styled components.
+customHeaders:
+  - pattern: '**'
+    headers:
+      - key: 'Strict-Transport-Security'
+        value: 'max-age=31536000; includeSubDomains'
+      - key: 'X-Frame-Options'
+        value: 'SAMEORIGIN'
+      - key: 'X-XSS-Protection'
+        value: '1; mode=block'
+      - key: 'X-Content-Type-Options'
+        value: 'nosniff'
+      - key: 'Content-Security-Policy'
+        value: "default-src 'self'; img-src * data:; script-src 'self' 'unsafe-inline' https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-chtml.js; style-src 'self' 'unsafe-inline' static:; connect-src orcid.org api.dev.nva.aws.unit.no api.sandbox.nva.aws.unit.no api.e2e.nva.aws.unit.no api.nva.unit.no api.test.aws.unit.no data:; font-src 'self' cdn.jsdelivr.net data:"
```

Comment:
usikker på img-src: * data:

Det betyr at vi kan hente inn bilder fra hvor som helst. Gjør vi det egentlig?

Reply:
Preview av bilde som er lastet opp på et resultat vises gjennom en presigned URL mot S3 iaf. Det er vel primært der bildene våre kommer fra, mener jeg.
Ellers er det mange som har ønske om profilbilde for alle brukere da, men det forutsetter at vi får en god bilde-server vi kan bruke fra Cristin, i stedet for å lage bilder basert på base64 strings som vi gjør noen få plasser nå

Reply:
Jeg må åpne opp for s3 buckets uansett, å sette s3 bucket fra eu-west-1 stenger ikke for så mye egentlig.